contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -23,11 +23,22 @@
 from opencensus.ext.azure.common.protocol import (
     Data,
     Envelope,
+    ExceptionData,
     RemoteDependency,
     Request,
 )
 from opencensus.ext.azure.common.storage import LocalFileStorage
 from opencensus.ext.azure.common.transport import TransportMixin
+from opencensus.ext.django.middleware import (
+    ERROR_MESSAGE,
```

Comment:
We don't want to be taking a dependency on `django` integration. Please use `attributes_helper.COMMON_ATTRIBUTES['ERROR_NAME']` directly.

Reply:
Done